pfunk/changepints.py

```diff
@@ -0,0 +1,78 @@
+#
+# Changepoint detection
+#
+# This file is part of Pints Functional Testing.
+#  Copyright (c) 2017-2019, University of Oxford.
+#  For licensing information, see the LICENSE file distributed with the Pints
+#  functional testing software package.
+#
+import numpy as np
+import ruptures as rpt
+
+class ChangePints:
+    """
+    Methods for changepoint detection
+
+    Example usage:
+    >>> ChangePints().data(signal).within_threshold()
+    False
+    """
+
+    def __init__(self, model="rbf", penalty=3):
+        """
+        Creates ChangePints object
+
+        :param str model: model (default rbf) to use, same as options in :class:`ruptures.detection.Pelt`
```

Comment:
We've been doing these lists manually, e.g. https://github.com/pints-team/functional-testing/blob/master/pfunk/_io.py#L113-L129

Maybe we should change this?

Reply:
Worth opening an issue in PINTS perhaps, if we don't already have one